src/modules/auth/index.js

```diff
@@ -170,6 +172,22 @@ export default class AuthModule extends Module {
 		});
 	}
 
+	notifyOnExpiredSession () {
+		const difference = moment(this.UserInfo.exp).valueOf() - moment.now();
+		const that = this;
+		
```

Comment:
I don't think this is correct.

The session can be prolonged by the user activity - which means that this method needs to be called periodically and do the check on the userinfo everytime it "feels" that the session is going to expire or is expired.

Only if that's correct, then the warning is displayed ... and it has to be done in a way that the user is not getting multiple alerts at one; which means that there needs to be some kind of expiration of the alert set logically to align with a next step.

Reply:
As Robin told me there is no way to prolong user activity so I thought that this is alright.

Reply:
Nope, it is not ... since the prolonging will be (obviously) added to SeaCat Auth soon.

Reply:
Also, I would like to avoid moments since we want to depart from them, correct?